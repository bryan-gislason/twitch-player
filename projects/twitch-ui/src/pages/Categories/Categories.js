import { useTranslation } from "@twitch-player/i18n";
import React, { memo } from "react";
import AllCategories from "../../components/AllCategories";

const Categories = () => {
  const { t } = useTranslation();

  return <AllCategories heading={t("pages.categories.mainHeading")} />;
};

export default memo(Categories);
