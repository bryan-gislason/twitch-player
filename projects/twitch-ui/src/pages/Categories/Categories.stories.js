import { storiesOf } from "@storybook/react";
import { createClient } from "@twitch-player/data";
import React from "react";
import { ApolloProvider } from "react-apollo-hooks";
import Main from "../../components/Main";
import Categories from ".";

storiesOf("Pages|Categories", module).add("default", () => (
  <ApolloProvider client={createClient()}>
    <Main>
      <Categories />
    </Main>
  </ApolloProvider>
));
