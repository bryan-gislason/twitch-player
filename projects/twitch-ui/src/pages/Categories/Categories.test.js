import React from "react";
import { render } from "../../testing";
import Categories from "./Categories";

jest.mock("../../components/AllCategories", () => "mock-all-categories");

describe("Categories", () => {
  let instance;

  beforeEach(() => {
    instance = render(<Categories />);
  });

  it("renders the page", () => {
    expect(instance.asFragment()).toMatchSnapshot();
  });
});
