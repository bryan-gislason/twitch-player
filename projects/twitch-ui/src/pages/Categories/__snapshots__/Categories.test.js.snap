// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories renders the page 1`] = `
<DocumentFragment>
  <mock-all-categories
    heading="Categories"
  />
</DocumentFragment>
`;
